docs/tutorial-code/generate_models.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+import sys
+from pathlib import Path
+
+if __name__ == "__main__":
+    for tutorial in Path(__file__).parent.iterdir():
+        if (tutorial / "generate_models.py").exists():
+            subprocess.call([sys.executable, str(tutorial / "generate_models.py")])
```

Comment:
Do you know if this function will ensure the individual `generate_models` are executed in the right order, given that they depend on each other? I'm asking because `glob` does not preserve the order and I remember seen a paper where there was a lot of trouble because the order in which files were scanned by the code was different in linux and windows. 

I would suggest to make sure the order is the one you want always by using `sorted` or similar function, so you do not rely on possible implementation details of `iterdir`.